modules/security-understanding-containers.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-understanding.adoc
+
+[id="security-understanding-containers_{context}"]
+= What are containers?
+
+Containers package an application and all its dependencies into a single image
+that can be promoted from development, to test, to production, without change.
+A container might be part of a larger application that works closely with other
+containers. 
+
+Containers provide consistency across environments and multiple deployment
+targets: physical servers, virtual machines (VMs), and private or public cloud.
+
+Some of the benefits of using containers include:
+
+// image::whatarecontainers.png["What Are Containers?", align="center"]
+
+[options="header",frame="all",grid="none"]
```

Comment:
I'd remove the `frame="all",grid="none"` options. We'd want all of our tables to look alike.